media/js/fonts.js

```diff
@@ -0,0 +1,158 @@
+(function(win, doc, $) {
+    /*
+        Font Loading
+    */
+
+    if(window.waffle && window.waffle.flag_is_active('web_font_loader')) {
+
+        // what fonts are we loading?
+        var fonts = {
+            "1" : {
+                "name" : "Open Sans Light",
+                "class" : "ffo-opensanslight",
+                "varient" : [
+                    {"weight" : "normal"},
+                    {"weight" : "bold"},
+                    {"style" : "italic"}
+                ]
+            },
+            "2" : {
+                "name" : "Open Sans",
+                "class" : "ffo-opensans",
+                "varient" : [
+                    {"weight" : "normal"},
+                    {"weight" : "bold"},
+                    {"style" : "italic"}
+                ]
+            } /*,
+            "3" : {
+                "name": "Open Sans Extra Bold",
+                "class" : "ffo-opensansextrabold",
+                "varient" : [
+                    {"weight" : "normal"}
+                ]
+            }*/
+        };
+
+        // load the observer plug in
+        $.getScript('/media/js/fontfaceobserver-min.js')
+        .done(function() { ffoStart(); })
+        .fail(function() { ffoFinished(); });
+
+        // to collect the results of the observers as they return
+        var fontsLoaded = {};
+
+        // timout for all observers
+        var ffoTimeout = 3000;
+
+        // kick off font loading
+        function ffoStart() {
+            ffoLoad();
```

Comment:
Actually, we can keep it, if it's for mental/maintenance reasons.  That might be good.